internal/compiler/typeregister.rs

```diff
@@ -31,6 +31,8 @@ pub const RESERVED_LAYOUT_PROPERTIES: &[(&str, Type)] = &[
     ("padding-right", Type::LogicalLength),
     ("padding-top", Type::LogicalLength),
     ("padding-bottom", Type::LogicalLength),
+    ("spacing-horizontal", Type::LogicalLength),
+    ("spacing-vertical", Type::LogicalLength),
```

Comment:
This is technically a breaking change if someone declared property in their code.

We reserved the padding property on all elements because we thought it might make sense on other item as well.
But spacing IMHO only makes sense in a layout.  So i'd keep this property only for the GridLayout and declare them for GridLayout in builtins.slint

Reply:
Done. 

I hadn't yet grasp that adding something here would impact all elements.